### ISSUE TITLE
Pick an email marketing system for signup form

### ISSUE THREAD

**Issue by Rabadash8820** [DerploidEntertainment/Website#10]
Resources listing the best CRM/email marketing tools out there:

1. [PCMag: The Best Email Marketing Software for 2022](https://www.pcmag.com/picks/the-best-email-marketing-software)
2. [wpbeginner: 7 Best Email Marketing Services for Small Business (2022)](https://www.wpbeginner.com/showcase/best-email-marketing-services/)
3. [Email Tool Tester: Compare the Top 5 Newsletter Services](https://www.emailtooltester.com/en/newsletter-software-comparison/)
4. [Influencer Marketing Hub: 11 of the Best Email Marketing Services for Small Businesses](https://influencermarketinghub.com/email-marketing-services-for-small-businesses/#toc-3)
5. [Website Builder Expert: 6 Best Email Marketing Services for Small Business](https://www.websitebuilderexpert.com/email-marketing/small-business-services/)
6. [Fit Small Business: 6 Best Email Marketing Software for Small Businesses in 2022](https://fitsmallbusiness.com/best-tools-for-email-marketing/)
7. [QuickSprout: Top 10 Best Email Marketing Services](https://www.quicksprout.com/best-email-marketing-services/)
8. [isitwp: 8 Best Email Service Providers](https://www.isitwp.com/best-email-marketing-services-compared/#undefined)
9. [BenchmarkOne: 10 Best Email Marketing Tools for Small Business](https://www.benchmarkone.com/blog/email-marketing-tools-for-small-business/)

**Comment by Rabadash8820** [comment]
Based on the above resources, the tools with the most recommendations (in desc order) were:

1. MailChimp (8)
2. Constant Contact (8)
3. SendinBlue (8)
4. GetResponse (6)
5. AWeber (6)

So these are the ones I took the time to dig into on this [comparison spreadsheet](https://docs.google.com/spreadsheets/d/1K5NZaidU1mLVpwy6aiwVstOg1ZRuSqRNEKVsLj6kw80/edit#gid=0) on Google Drive. I looked at cost, max contacts, email send limit, customer support, and general user-friendliness and feature-completeness according to those articles (user segmentation, email scheduling/automation, number of templates, etc.). It ultimately boils down to what we're willing to pay for, because different services are more worthwhile at different cost tiers. I can't imagine we'll ever have more than a few hundred contacts, or send more than one or two messages to each of them in a month, so I don't really want a recurring monthly fee for this small use case. Based on the above research the best _free tier_ service seems to be SendinBlue. So yeah, I'm voting

**SendinBlue**

@asdkiller and I will discuss and close this issue once we reach a decision.
